docs/examples/notebooks/audio_decoder.ipynb

```diff
@@ -0,0 +1,486 @@
+{
```

Comment:
@swetha097  : Please remove all the output data from this PR. The user is supposed to run the notebook and get output

Reply:
Done - removed all the cell outputs